File: help/eng_gb/exercicios_da_ficha_7/Ficha07_006/Ficha07_006.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha07_006 {

    //Main Function
    static void Main() {
        Console.Write("Number of terms : " );
        int n = int.Parse(Console.ReadLine());
        int product = 1 ;
        for( int i = 1; i < n + 1; i = i + 1 ) {
            int f = 2 * i - 1 ;
            product *=f;
        }
        Console.Write("Result = " + product );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Fri, 2022 Sep 30 22:37

File: help/eng_gb/exercicios_da_ficha_7/Ficha07_006/Ficha07_006.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Number of terms : " );// print text
    n = parseInt(prompt("Number of terms : " ,"0") );
    
    document.write(n + "\n");// print input 
    var product = 1 ;
    for( var i = 1; i < n + 1; i = i + 1 ) {
        var f = 2 * i - 1 ;
        product *=f;
    }
    document.write("Result = " + product );
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Fri, 2022 Sep 30 22:37
